.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,7 +8,7 @@
  - [ ] document any changes to public APIs
  - [ ] post benchmark scores
  - [ ] manually test the debug page
- - [ ] tagged `@mapbox/studio` and/or `@mapbox/map-design-team` if this PR includes style spec or visual changes
+ - [ ] tagged `@mapbox/map-design-team` `@mapbox/gl-native` `@mapbox/static-apis` if this PR includes style spec API or visual changes
```

Comment:
`@mapbox/gl-native` is listed below so it shouldn't be included here, right?

Reply:
sure, makes sense, thanks for the catch - it's unlikely this box would be checked and not the next one (i.e. style spec API or visual change that does not require a native port / shader change)